Tests/SwiftDocCTests/Infrastructure/Symbol Link Resolution/AbsoluteSymbolLinkTests.swift

```diff
@@ -510,6 +499,16 @@ class AbsoluteSymbolLinkTests: XCTestCase {
                 basePathComponents: [(name: 'func()', suffix: (idHash: '6ijsi'))]
             }
             """,
+            // doc://org.swift.docc.example/documentation/SideKit/UncuratedClass:
+            """
+            {
+                bundleID: 'org.swift.docc.example',
+                module: 'SideKit',
+                topLevelSymbol: (name: 'UncuratedClass', suffix: (none)),
+                representsModule: false,
+                basePathComponents: []
+            }
+            """,
```

Comment:
These items are sorted, so this one moved now that the path doesn't contain "NonExistent" anymore.